Formula/qemu.rb

```diff
@@ -61,7 +62,6 @@ def install
       --enable-vde
       --extra-cflags=-DNCURSES_WIDECHAR=1
       --disable-sdl
-      --disable-gtk
```

Comment:
We probably want to keep passing this on macOS to avoid opportunistic linkage.

Reply:
OK, can add it the bottom